conf/README.md

```diff
@@ -41,6 +41,7 @@ run it belongs here.
 * `username` - Kubeadmin username
 * `password_location` - Filepath (under the cluster path) where the kubeadmin password is located
 * `log_dir` - Directory where logs are placed
+* `logs_url` - URL where will be available logs for remote access, used for Jenkins runs and configured by Jenkins
```

Comment:
rewrite - URL where the logs will be available for remote access,

Reply:
Updated, I was thinking about the word _rewrite_, but it is used also in the *Environment* section, where it doesn't rewriting anything in fact. So I change the wording, but without the _rewrite_.

Reply:
i didn't mean to include word rewrite but re-word the description :)